server/src/test/java/org/apache/iotdb/db/conf/adapter/IoTDBConfigDynamicAdapterTest.java

```diff
@@ -66,7 +66,7 @@ public void addOrDeleteStorageGroup() throws ConfigAdjusterException {
     for (int i = 1; i < 1000000; i++) {
       try {
         IoTDBConfigDynamicAdapter.getInstance().addOrDeleteStorageGroup(1);
-        memTableNum += IoTDBDescriptor.getInstance().getConfig().getMemtableNumInEachStorageGroup();
+        memTableNum += IoTDBDescriptor.getInstance().getConfig().getConcurrentWritingTimePartition() * 4 + 1;
```

Comment:
The same issue, set 4 as const and a reference from addOrDeleteStorageGroup()

Reply:
Sure~ I will fix it!